ios/Classes/NativeImagesPlugin.m

```diff
@@ -1,37 +0,0 @@
-#import "NativeImagesPlugin.h"
-
-#if !__has_feature(objc_arc)
-#error ARC must be enabled!
-#endif
-
-@interface NativeImagesPlugin ()
-@end
-
-@implementation NativeImagesPlugin
```

Comment:
should the name of the plugin be a bit looser (so that you could potentially add APIs for loading fonts etc as well) or would you create a different plugin for fonts?

Seems like the former would be less work.